src/utils/dayPeriod.js

```diff
@@ -0,0 +1,8 @@
+export default function getDayPeriod(time) {
+  const hours = new Date(time)
+    .getHours();
+  const startOfDay = 6;
+  const endOfDay = 18;
```

Comment:
Since we operating coordinates - consider usage of https://github.com/mourner/suncalc or something like this instead of hardcoding values